examples/notation.rst

```diff
@@ -15,7 +15,8 @@ typically) by letting :math:`\Omega = \bigcup_{e=1}^{N_e}\Omega_e`, with :math:`
 disjoint elements. For most examples we use unstructured meshes for which the elements
 are hexahedra (although this is not a requirement in libCEED).
 
-The physical coordinates are denoted by :math:`\mathbf{x}=(x,y,z)\in\Omega_e`,
+The physical coordinates are denoted by
+:math:`\bm{x}=(x,y,z) \equiv (x_0,x_1,x_2) \in\Omega_e`,
 while the reference coordinates are represented as
-:math:`\boldsymbol{X}=(X,Y,Z) \equiv (X_1,X_2,X_3) \in\mathbf{I}=[-1,1]^3`
+:math:`\bm{X}=(X,Y,Z) \equiv (X_0,X_1,X_2) \in\bm{I}=[-1,1]^3`
```

Comment:
Do we need to dedicate `\bm I` for this? It seems like we most always write out `[-1,1]^d` anyway, in which case that named symbol is unnecessary.

Reply:
I will remove the bold font for this, but I like to denote intervals. I think it's neater, and we may have to re-refer to it later. I'll check if we actually do in this case.

Reply:
It looked like it repeated `\bm I = [-1, 1]^3` any time the interval was used, in which case the `I` was superfluous.

Reply:
I thought it was redefined only when necessary because different, for instance in the area problem `I=[-1,1]^2`. But I may have missed some repetitions. I'll look for those